lxd/instance/instance_utils.go

```diff
@@ -111,6 +112,14 @@ func ValidConfig(sysOS *sys.OS, config map[string]string, expanded bool, instanc
 		return fmt.Errorf("nvidia.runtime is incompatible with privileged containers")
 	}
 
+	// Validate pinning strategy when limits.cpu specifies static pinning.
+	cpuPinStrategy := config["limits.cpu.pin_strategy"]
+	cpuLimit := config["limits.cpu"]
+	staticPinning, _ := validate.IsStaticCPUPinning(cpuLimit)
+	if !expanded && staticPinning && cpuPinStrategy == "auto" {
+		return fmt.Errorf("CPU pinning specified, but pinning strategy is set to 'auto'")
```

Comment:
Please can you use backticks for the fmt.Errorf here and then use double quotes to wrap "auto".

Reply:
Sure thing!